### PR TITLE
add more functionality to status command, add reset command, clean up…

### PR DESCRIPTION
**Pull request by CwickWalt**
… code

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -120,8 +128,19 @@ function provideStatus(channel, option, value) {
...
+    date = `${value}`;
+    channel.send(`Date was updated to: ${date}`);
+}
+
+/**
+ * Update the time of the tournament
+ * @param {TextChannel | DMChannel | GroupDMChannel} channel 
+ * @param {string} value 
+ */
+
```
Comment (RiceOnly) [inline bot.js]: Clear this line

File: bot.js [hunk 2]
```diff
@@ -232,14 +275,17 @@ function availableCommands(channel, option) {
 function statusUtil(channel, option, value) {
     if (!option) {
         provideStatus(channel, option);
-        provideDate(channel, option);
+        provideDateAndTime(channel, option);
```
Comment (RiceOnly) [inline bot.js]: Also missing another param

File: bot.js [hunk 3]
```diff
@@ -130,8 +149,30 @@ function updateDate(channel, value) {
...
  * @param {string} value
  */
-function provideDate(channel, option, value) {
-    channel.send(date);
+function provideDateAndTime(channel, option, value) {
+    if(date && !time) {
+        displayDate = `__Tournament Date__: **${date}**. A **time** has not been set. Use *ss~status time* <**00:00**> to update the time.`
+        channel.send(displayDate);
+    } else if(time && !date) {
+        displayDate = `__Tournament Date__: **${time}**. A **date** has not been set. Let's be honest, it's probably on Sunday :/, but use *ss~status date* <**MM/YYYY**> to update the date anyway because <:FeelsDumbMan:471898989206306816>`
```
[the remark this answers is not in the document]
Comment (CwickWalt) [reply bot.js]: the bot can use all emotes in discord regardless of channel.

File: bot.js [hunk 4]
```diff
@@ -206,17 +247,19 @@ function showParticipants(channel, option) {
  */
 function availableCommands(channel, option) {
     if (!option) {
-        channel.send(`Available commands:
+        channel.send(`\`\`\`Available commands:
```
[the remark this answers is not in the document]
Comment (CwickWalt) [reply bot.js]: discord markup uses ``` text ``` to make codeblocks, which makes the text look distinctively different from normal text. To add the markup, i have to use 4 back ticks ````, but i need to escape 3 of them, hence `\`\`\`.

[the remark this answers is not in the document]
Comment (RiceOnly) [reply bot.js]: Makes sense, it looks nicerrrr with the codeblocks

**Pull request merged**